tests/testthat/test-test-server.R

```diff
@@ -693,12 +693,37 @@ test_that("It's an error to pass arguments to a server", {
   expect_error(testServer(test_path("..", "test-modules", "06_tabsets"), {}, args = list(an_arg = 123)))
 })
 
+# Provided an instance of an R6 object, returns a list with `methods` and
+# `fields`. `methods` contains a character vector of names of public methods.
+# `fields` is a character vector of public fields. Any active bindings are
+# considered `fields`.
+get_mocked_publics <- function(instance) {
+  generator <- get(class(instance)[[1]])
+  publics <- ls(mock_session)
+  actives <- names(generator$active)
+  # Active bindings are considered fields.
+  methods_or_fields <- publics[!(publics %in% actives)]
+  methods <- character(0)
+  fields <- c(character(0), actives)
```

Comment:
Any particular reason to not do:

```
fields <- actives
```

Reply:
`fields` might then be `NULL` instead of an empty character vector and I thought there might be some value in the field being type stable